build/configs/rtl8720e/hello/defconfig

```diff
@@ -618,7 +618,7 @@ CONFIG_NET_UDP_TTL=255
 CONFIG_NET_UDPLITE=y
 CONFIG_NET_TCP=y
 CONFIG_NET_TCP_TTL=255
-CONFIG_NET_TCP_WND=14600
+CONFIG_NET_TCP_WND=26280
```

Comment:
Could you modify this in other defconfigs like tc and loadable_apps as well?

Reply:
okok no problem